app/main.py

```diff
@@ -16,4 +16,19 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+
+def _variable_sorter(*args) -> dict:
+
+    result = {"mutable": [], "immutable": [], }
+
+    for var_name in [_ for _ in args[0] if not _.startswith("_")]:
+        if (type(eval(var_name)) is dict or type(eval(var_name)) is list
+                or type(eval(var_name)) is set):
+            result["mutable"].append(eval(var_name))
+            continue
+        result["immutable"].append(eval(var_name))
+
+    return result
+
+
+sorted_variables = _variable_sorter(dir())
```

Comment:
you made everything very difficult

Reply:
![image](https://github.com/mate-academy/py-mutable-immutable/assets/93856423/0e403d36-4d30-4f99-92e3-4c10ef49a41a)

Reply:
I know, but I wanted to unify this function not only for these eight arguments, but also in case of passing more arguments. Ok, I'll simpify :)